contrib/container/Dockerfile

```diff
@@ -74,8 +74,9 @@ RUN apk add --no-cache \
     # MySQL / MariaDB client
     mariadb-client mariadb-connector-c \
     && \
-    # fonts
-    apk --update --upgrade --no-cache add fontconfig ttf-freefont font-noto terminus-font && fc-cache -f
+    # font support
+    apk --update --upgrade --no-cache add fontconfig ttf-freefont font-terminus font-inconsolata font-dejavu font-noto font-noto-cjk font-noto-extra \
```

Comment:
What purpose servers `font-inconsolata font-dejavu`? `font-noto-cjk` should be enough for Chinese / Korean / japanese

Reply:
Agreed, I'll update the PR and remove those. I was initially just following the recommendations from the alpine font support page.